cetmix_tower_server/__init__.py

```diff
@@ -1,2 +1,83 @@
+# pylint: disable=E8103
+
+import logging
+
 from . import models
 from . import wizards
+
+_logger = logging.getLogger(__name__)
+
+
+def _post_init_hook(cr, registry):
```

Comment:
Post_init_hook are the functions that will execute after the module installation. Maybe in your case you need to use migrations to work with the exists data

Reply:
@tendil why do you resolve a dialog without a response that is also directed to the author of the PR?